src/assets/scss/base/_settings.scss

```diff
@@ -18,10 +18,14 @@ $primary-gradient: #53b1d4;
 $secondary-gradient: #9edbdf;
 $tertiary-gradient: #d2c4db;
 
+body {
```

Comment:
This is probably my fault from yesterday but just keep all the body stylings in one place, so move these to _global.scss